src/System.Net.NameResolution/src/System/Net/DNS.cs

```diff
@@ -112,8 +112,11 @@ internal static IPHostEntry InternalGetHostByAddress(IPAddress address, bool inc
 
                     if (NetEventSource.Log.IsEnabled())
                     {
-                        NetEventSource.Exception(NetEventSource.ComponentType.Socket, "DNS",
-                        "InternalGetHostByAddress", new InternalSocketException(errorCode, nativeErrorCode));
+                        NetEventSource.Exception(
+                            NetEventSource.ComponentType.Socket, 
+                            "DNS",
+                            "InternalGetHostByAddress", 
```

Comment:
I could do that for "InternalGetHostByAddress", but not for "DNS", which would change casing to "Dns".  That might be OK, but I'd rather not fold that into this change.